src/index.ts

```diff
@@ -95,3 +95,11 @@ export * from "./properties";
 export * from "./analysis/static/Target";
 export * from "./analysis/static/TargetPool";
 export * from "./analysis/static/targetUtil";
+
+// Events
```

Comment:
They are not events but related to events

```suggestion
// Event System
```